actions/rules/action_field_set_test.go

```diff
@@ -23,63 +22,77 @@ type ActionFieldSetSuite struct {
 	gormtestsupport.DBTestSuite
 }
 
-func createWICopy(ID uuid.UUID, state string, boardcolumns []string) workitem.WorkItem {
+func createWICopy(wi workitem.WorkItem, state string, boardcolumns []interface{}) workitem.WorkItem {
 	var wiCopy workitem.WorkItem
-	wiCopy.ID = ID
-	fields := map[string]interface{}{
-		"system.state":        state,
-		"system.boardcolumns": boardcolumns,
+	wiCopy.ID = wi.ID
+	wiCopy.SpaceID = wi.SpaceID
+	wiCopy.Type = wi.Type
+	wiCopy.Number = wi.Number
+	wiCopy.Fields = map[string]interface{}{}
+	for k := range wi.Fields {
+		wiCopy.Fields[k] = wi.Fields[k]
 	}
-	wiCopy.Fields = fields
+	wiCopy.Fields["system.state"] = state
+	wiCopy.Fields["system.boardcolumns"] = boardcolumns
 	return wiCopy
 }
 
 func (s *ActionFieldSetSuite) TestActionExecution() {
-	fixture := tf.NewTestFixture(s.T(), s.DB, tf.CreateWorkItemEnvironment(), tf.WorkItems(2))
-	require.NotNil(s.T(), fixture)
-	require.Len(s.T(), fixture.WorkItems, 2)
+	fxt := tf.NewTestFixture(s.T(), s.DB, tf.CreateWorkItemEnvironment(), tf.WorkItems(2))
+	require.NotNil(s.T(), fxt)
+	require.Len(s.T(), fxt.WorkItems, 2)
 
 	s.T().Run("sideffects", func(t *testing.T) {
-		fixture.WorkItems[0].Fields["system.state"] = "new"
-		fixture.WorkItems[0].Fields["system.boardcolumns"] = []string{"bcid0", "bcid1"}
-		newVersion := createWICopy(fixture.WorkItems[0].ID, "open", []string{"bcid0", "bcid1"})
-		contextChanges, err := fixture.WorkItems[0].ChangeSet(newVersion)
+		fxt := tf.NewTestFixture(s.T(), s.DB, tf.CreateWorkItemEnvironment(), tf.WorkItems(2))
```

Comment:
Beware, that you're introducing the `s.T()` problem in sub-tests here again.

Reply:
Done.